components/webgpu/dom_messages.rs

```diff
@@ -291,9 +295,24 @@ pub enum WebGPURequest {
         device_id: id::DeviceId,
         command_encoder_id: id::CommandEncoderId,
     },
+    // Render Pass
```

Comment:
I think these should be tripe slash

Reply:
Same reasoning as https://github.com/servo/servo/pull/32665#discussion_r1665126864, as this comment describes section of enum not only one variant